parts/inc/utf8

```diff
@@ -182,10 +182,10 @@ __UNDEFINED__ REPLACEMENT_CHARACTER_UTF8  "\xDD\x72\x72\x70"
 #  error Unknown character set
 #endif
 
-#if { VERSION < 5.31.4 }
-        /* Versions prior to this accepted things that are now considered
-         * malformations, and didn't return -1 on error with warnings enabled
-         * */
+#if { VERSION < 5.35.10 }
+        /* Versions prior to 5.31.4 accepted things that are now considered
+         * malformations, and didn't return -1 on error with warnings enabled.
+         * Versions before 5.35.10 dereferenced empty input without checking */
```

Comment:
wow so this is not limited to versions `>= 5.35.9` has it looks like the older versions have some different issues...

Reply:
probably an issue from `utf8n_to_uvchr`